bin/RegisterTritonDialects.h

```diff
@@ -36,6 +38,9 @@ inline void registerTritonDialects(mlir::DialectRegistry &registry) {
   mlir::triton::registerConvertTritonToTritonGPUPass();
   mlir::triton::registerConvertTritonGPUToLLVMPass();
   mlir::triton::registerConvertNVGPUToLLVMPass();
+  if (!::triton::tools::getBoolEnv("TRITON_DISABLE_LINE_INFO")) {
+    mlir::registerLLVMDIScope();
+  }
```

Comment:
how is this related? Is there any reason to not always call `mlir::registerLLVMDIScope();`?

Reply:
Yeah, I think this was silly on my part. It should register the pass always in the bin tools I think.